client/client_update_test.go

```diff
@@ -1309,6 +1311,25 @@ func testUpdateRemoteKeyRotated(t *testing.T, targetsRole string) {
 		msg, reflect.TypeOf(err))
 }
 
+// Helper function that takes a signedRoot, and signs it with the provided keys and only these keys.
+// Then serializes this to bytes and updates the swizzler with it, and updates the snapshot and
+// timestamp too so that the update won't fail due to a checksum mismatch.
+func signSerializeAndUpdateRoot(t *testing.T, signedRoot data.SignedRoot,
+	serverSwizzler *testutils.MetadataSwizzler, keys []data.PublicKey) {
+
+	signedObj, err := signedRoot.ToSigned()
+	require.NoError(t, err)
+	// sign with the first two keys
```

Comment:
Nice catch, thanks!  It was copied from the previous test, so the comment is no longer applicable to t he general case.